sdk/table/azure-table/azure/table/_generated/py.typed

```diff
@@ -0,0 +1 @@
+# Marker file for PEP 561.
```

Comment:
Put a copy of this file in ` sdk/table/azure-table/azure/table`. This enables mypy on your package.